File: src/utils/response.js
// 응답 봉투는 성공/실패 무관하게 항상 { code, message, data } 형태로 온다.
// HTTP 200이어도 실패일 수 있으므로 code로만 성공을 판정한다.
// (예: 플래그 오답은 200 OK + code: "INCORRECT_FLAG")
export function isSuccess(envelope) {
  return envelope?.code === "SUCCESS";
}

File: mock-backend/api/response.py
"""README.md 0-2절: 모든 응답은 { code, message, data } 3개 키 고정."""

from rest_framework.response import Response


def success(data=None, message="성공", status=200):
    return Response({"code": "SUCCESS", "message": message, "data": data}, status=status)


def fail(code, message, status=400, data=None):
    return Response({"code": code, "message": message, "data": data}, status=status)
